packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -805,7 +820,7 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with Restoratio
   FocusNode get _effectiveFocusNode => widget.focusNode ?? (_focusNode ??= FocusNode());
 
   MaxLengthEnforcement get _effectiveMaxLengthEnforcement => widget.maxLengthEnforcement
-    ?? LengthLimitingTextInputFormatter.inferredDefaultMaxLengthEnforcement;
+    ?? LengthLimitingTextInputFormatter.getDefaultMaxLengthEnforcement();
```

Comment:
like the new name :+1:

Reply:
Is this related to the deprecation at all or just an improvement?

Reply:
It's an improvement, that the style guide say we preferred `getXXX` for method.